index.html

```diff
@@ -908,18 +938,45 @@ <h3>Layout</h3>
       <section id="progression">
         <h3>Progression</h3>
         <p id="r_save_progresssion" class="req-set cc-extended">
-          User agents should provide the option to save this progression in the publication and returns the user to
-          their last location the next time they open the publication.
+          User agents should provide the option for the user to save their progression in the publication and return the user to
+          the last location they saved the next time they open the publication.
         </p>
         <ul class="use-cases">
           <li>
-            Filbert is reading a comic as a Web Publication. In his user agent, he marks a specific page to return to
-            and exits the user agent. When he returns the next day, he opens the Web Publication and continues reading
-            from the location he marked.
+            Mateusz runs a Dungeons and Dragons campaign.  He would like to be able to mark off specific sections of
+            different rulebooks and share those bookmarks so that players in his game can quickly reference the
+            material.
+          </li>
+          <li>
+            Aika is reading a novel on her 9 inch tablet, but later switches to her 5 inch phone.  She would like to
+            be able to resume reading from the same point where she left off, given that she may not be using the same
+            user agent on each device that contains some mechanism that would sync the content position.
           </li>
         </ul>
         <ul class="wpub-xref">
           <li><a href="https://w3c.github.io/wpub/#short-description-0">Reading State &gt; Short Description</a></li>
+          <li><a href="https://w3c.github.io/wpub/#locators">Web Publication Locators</a></li>
+        </ul>
+      </section>
+
+      <section id="multiformatsync">
```

Comment:
Added this reading state requirement. It comes directly from the spec, and I believe it is related to this idea of page numbering and bookmarking.